services-speech/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'java-library'
+
+dependencies {
+    compile project(":services-core")
```

Comment:
Follow https://github.com/mapbox/mapbox-java/pull/737 to change the `compile` to `implemantation/api`